docs/source/format/FlightSql.rst

```diff
@@ -149,6 +149,12 @@ the ``type`` should be ``ClosePreparedStatement``).
     When used with DoPut: execute the query and return the number of
     affected rows.
 
+``CommandStatementIngest``
+    Execute a bulk ingestion.
+
+    When used with DoPut: load the stream of Arrow record batches into
+    the specified target table and return the number of rows ingested.
```

Comment:
The Flight SQL spec should explain how the caller can retrieve the number of rows ingested. Apparently it's encoded in the `app_metadata`? Is it using protobuf? JSON?

Reply:
I agree this was not made very clear. Yes there is a Flight SQL protobuf `DoPutUpdateResult` message encoded in the `app_metadata`. I've just pushed some updates to documentation and comments to clarify this.

Reply:
```suggestion
    the specified target table and return the number of rows ingested
    via a `DoPutUpdateResult` message.
```